zerver/outgoing_webhooks/generic.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from typing import Any, Dict, Text, Tuple, Callable, Mapping
+import json
+from requests import Response
+from zerver.models import UserProfile
+
+from zerver.lib.outgoing_webhook import OutgoingWebhookServiceInterface
+
+class Generic(OutgoingWebhookServiceInterface):
```

Comment:
The way I understand it, this class is an implementation of an interface, so I think it's slightly strange to call it `GenericInterface`.

What exactly is this class?  Is it a `GenericService`?  Is it a `GenericOutgoingWebhookService`?